packages/orbit-components/src/Collapse/Collapse.stories.tsx

```diff
@@ -116,25 +127,17 @@ export const WithActions = ({ label }) => {
         <Checkbox label="Train" value="three" />
       </ChoiceGroup>
     </Collapse>
-  );
-};
+  ),
 
-WithActions.story = {
-  name: "With actions",
-
-  parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+  args: {
+    label: "Transportation",
   },
 };
 
-WithActions.args = {
-  label: "Transportation",
-};
-
-export const MultipleCollapses = ({ label }) => {
-  return (
+export const MultipleCollapses: Story = {
```

Comment:
I can't interact with this story in the same way I do with this [one](https://kiwicom.github.io/orbit/?path=/story/collapse--multiple-collapses).

Reply:
The interaction is "blocked" by added switches in controls. From my observation, it's not possible to have switches for expanding and also have `expanded` function available by clicking on the arrow button.

Reply:
I believe this can be achieved by using `@storybook/preview-api`, could you take a look at [these docs](https://storybook.js.org/docs/writing-stories/args#setting-args-from-within-a-story)?

Reply:
There is perhaps a 4th option. What do you think about removing the arg for this particular case and relying on the UI user interaction? In the end, that's the regular use case (uncontrolled component).

Reply:
Do you mean removing `expanded` arg for multipleCollapse at all? yes, we can go this way, I just wanted to include as much args as possible, but it also makes sense to exclude that arg from this story. 🤔

Reply:
Should be changed. I used `api-preview` at least for single collapse components. :)